test/integration/pool.test.js

```diff
@@ -73,7 +73,7 @@ function attachMSSQLUniqueId(connection) {
 }
 
 describe(Support.getTestDialectTeaser('Pooling'), () => {
-  if (dialect === 'sqlite' || process.env.DIALECT === 'postgres-native') {
+  if (dialect === 'sqlite' || Support.getDialectName(process.env.DIALECT) === 'postgres-native') {
```

Comment:
```suggestion
  if (dialect === 'sqlite' || dialect === 'postgres-native') {
```

Reply:
```suggestion
  if (['sqlite','postgres-native'].includes(dialect)) {
```
This might be even nicer